src/components/AssetsPage/AssetsPage.test.jsx

```diff
@@ -1,25 +1,31 @@
 import React from 'react';
 import Enzyme from 'enzyme';
 
-import { AssetsPage, mapDispatchToProps, types } from './index';
+import AssetsPage, { types } from './index';
 import { assetActions } from '../../data/constants/actionTypes';
-// import { types } from './pageTypes';
 
 const { shallow } = Enzyme;
 
 let wrapper;
 
 const defaultProps = {
   assetsList: [],
-  courseDetails: {},
+  courseDetails: {
```

Comment:
TODO: We're defining this in many tests. I'd like to pull it out into some kind test constants file.